src/stylesheets/layouts/_concept.scss

```diff
@@ -0,0 +1,380 @@
+.solution {
+  background: linear-gradient($color-medium-blue 5%, $color-mint 90%);
+  @include flex;
+
+  &__wrapper {
+    position: relative;
+    @include flex;
+    width: 100%;
+    height: 100%;
+    max-width: 77rem;
+    margin: 0;
+    padding: 2.5rem 1.5rem;
```

Comment:
🔤

Reply:
Note on 🔤 - it's fine if mixins come _before_ your other alphabetized properties. Like so:

```
.item {
  @include flex;
  height: 100%;
  margin: 0;
  max-width: 77rem;
  padding: 2.5rem 1.5rem;
  width: 100%;
}
```

Reply:
That's important just because sometimes you have to override a style added by a mixin

Reply:
ok cool. resolved in pr#4